### ISSUE TITLE
Balance: Gaslighter and butterfly are too easy, Murmurs and Maze are too hard

### ISSUE THREAD

**Issue by SkylarkHYP** [db0/hypnagonia#415]
Act 1's balancing is very strange and unpredictable, as well as somewhat unfair, so I have some rudimentary suggestions on some of the things I have noticed, after playing other games like Slay The Spire now.

Gaslighter: It's comprehension is too low, my suggestion would to make it 1.5x or 2x more, as well as giving it courage and higher stress intents. Otherwise I can usually get an overcome on turns 2 to 3. (Free cards essentially)
i'd also like to mention that the gasligher understanding is WAY too good. As taking it on Act I  is a good counter for Narcisuss. My suggestion is to make it remove 5-10 doubt, instead of ALL doubt.

Butterfly: The butterfly also suffers from a low comprehension pool, not to mention the card reward is very good compared to the actual torment. Butterfly should scale 1 focus every turn, since it barely does any damage, and it's contentment is too manageable. 

Murmurs: You know how StS has "The Guardian", well essentially Murmurs is like its defensive form. With a higher health pool of around 40 and it applies confusion and can scale grudge, Murmurs should be an elite at least. (But I'm almost kidding), Murmurs probably just needs less comprehension,  Maybe switch murmurs and gaslighter's comprehension pools.

Maze: I HATE MAZE. THERE IS LITERLALLY NO ESCAPE FROM TAKING ANXIETY FROM IT. It is literally causing me real life anxiety. I know you were trying to replicate Nob, but Maze i way too OVERPOWERED. My suggestion is to make it so that every time you play and action on it you relax for like 1-2, or make maze and act II elite. Because OMG I almost always either die to maze or reach 50-70% of my max anxiety on 0 anxiety.

TLDR:
Gaslighter needs more health +buffs
Butterfly needs more health+buffs
Mumurs needs less health
Maze needs to be deleted from the game (Needs major nerfs)

Every rest of the torments I can agree are balanced.
